Description: Sort files in d directory when making tvalues.m2
 Otherwise, tvalues.m2 is not reproducible.
Author: Doug Torrance <dtorrance@piedmont.edu>
Bug: https://github.com/Macaulay2/M2/issues/2112
Applied-Upstream: 1.19, https://github.com/Macaulay2/M2/pull/2188
Last-Update: 2021-07-11

--- a/M2/Macaulay2/m2/tvalues-make.m2
+++ b/M2/Macaulay2/m2/tvalues-make.m2
@@ -6,7 +6,7 @@
 make = (srcdir) -> (
      outfile = openOut "tvalues.m2";
      srcdir = minimizeFilename srcdir;
-     scan(select(readDirectory srcdir, dir -> match("\\.dd?$",dir)), file -> (
+     scan(sort select(readDirectory srcdir, dir -> match("\\.dd?$",dir)), file -> (
 	       file = srcdir | file;
 	       txt := get file;
 	       if match("-- . typical value: ",txt) then (
